ts/Core/Renderer/SVG/SVGRenderer.ts

```diff
@@ -2474,6 +2471,20 @@ class SVGRenderer {
         );
 
     }
+
+    /**
+     * Re-align all aligned elements.
+     *
+     * @private
+     * @function Highcharts.SVGRenderer#align
+     * @return {void}
+     */
+    public align(): void {
+        let i = this.alignedObjects.length;
+        while (i--) {
```

Comment:
Since speed is not critical here (we're only doing a handful of elements), maybe we should go with the more compact `forEach` loop.